common/src/main/java/gov/cms/ab2d/common/repository/JobRepository.java

```diff
@@ -38,4 +40,9 @@
     @Query("UPDATE Job j SET j.status = 'SUBMITTED' WHERE j.jobUuid IN :jobUuids ")
     void resetJobsToSubmittedStatus(List<String> jobUuids);
 
+
+    @Modifying
```

Comment:
Does the method above -- `resetJobsToSubmittedStatus` need to be `Modifying` too then?

Reply:
Yes. It is also annotated with `@Modifying`